sentry-spring-boot-starter/build.gradle.kts

```diff
@@ -60,6 +61,8 @@ dependencies {
     testImplementation(kotlin(Config.kotlinStdLib))
     testImplementation(Config.TestLibs.kotlinTestJunit)
     testImplementation(Config.TestLibs.mockitoKotlin)
+    testImplementation(Config.TestLibs.mockWebserver)
+    testImplementation(Config.Libs.okhttp)
```

Comment:
`Config.Libs.okHttp` does not specify exact OkHttp version. As a result it loads MockWebServer 4.9.0, but OkHttp 3.x which causes later on compilation errors. 

In the `dependencyManagement` configuration in this file (search for `the<DependencyManagementExtension>().apply {`) add:

```
mavenBom(Config.Libs.okhttpBom)
```
To make sure correct version is used.

Reply:
Ah.. I ran into this issue and then it was suddenly gone.
Committed the changes in 713b11a4c95b0741d9f5cc34022c1811048c30c8